### ISSUE TITLE
Sim xss fuzz error for a11y strings

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/ohms-law#110]
Possibly related to #108, @zepumph could you please look into this?

```
ohms-law : xss-fuzz : load
Uncaught Error: Assertion failed: Undefined not allowed for Node key: accessibleLabel
Error: Assertion failed: Undefined not allowed for Node key: accessibleLabel
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/assert/js/assert.js:22:13)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/scenery/js/nodes/Node.js?bust=1521150780417:4614:19
    at u (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/sherpa/lib/lodash-4.17.4.min.js?bust=1521150780417:5:432)
    at Function.ru (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/sherpa/lib/lodash-4.17.4.min.js?bust=1521150780417:67:656)
    at OhmsLawScreenView.mutate (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/scenery/js/nodes/Node.js?bust=1521150780417:4612:9)
    at OhmsLawScreenView.Node [as constructor] (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/scenery/js/nodes/Node.js?bust=1521150780417:503:12)
    at OhmsLawScreenView.ScreenView [as constructor] (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/joist/js/ScreenView.js?bust=1521150780417:35:10)
    at new OhmsLawScreenView (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/ohms-law/js/ohms-law/view/OhmsLawScreenView.js?bust=1521150780417:52:16)
    at OhmsLawScreen.createView (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/ohms-law/js/ohms-law/OhmsLawScreen.js?bust=1521150780417:29:34)
    at OhmsLawScreen.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1521149673292/joist/js/Screen.js?bust=1521150780417:243:25)
Approximately 3/15/2018, 3:34:33 PM
```

**Comment by zepumph** [comment]
Thanks for this, oops.
